### ISSUE TITLE
assembleRelease error caused by hard-coded "/Users/christopherdro/Desktop/OSS/react-native-maps/" path for @christopherdro. 

### ISSUE THREAD

**Issue by joshuapinter** [react-native-maps/react-native-maps#3799]
# Bug report

## Summary

After installing this package, running `gradlew assembleRelease` fails with the following error message related to hardcoded paths for `christopherdro` user: 

```
> Task :react-native-maps:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-maps:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > 1 exception was raised by workers:
     com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
     AAPT: /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_custom.xml:63: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_custom.xml:71: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_custom.xml:79: error: resource style/TextAppearance.Compat.Notification.Info.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Info.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_narrow_custom.xml:66: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_narrow_custom.xml:74: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_narrow_custom.xml:82: error: resource style/TextAppearance.Compat.Notification.Info.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Info.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:36: error: resource style/TextAppearance.Compat.Notification.Title.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Title.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:45: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:56: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:68: error: resource style/TextAppearance.Compat.Notification.Line2.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Line2.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:90: error: resource style/TextAppearance.Compat.Notification.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_lines_media.xml:100: error: resource style/TextAppearance.Compat.Notification.Info.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Info.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_media_custom.xml:55: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_media_custom.xml:63: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.
     /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_media_custom.xml:71: error: resource style/TextAppearance.Compat.Notification.Info.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Info.Media) not found.
     error: failed linking file resources.
```

Running `gradlew clean` and then running `gradlew assembleRelease` works just fine.

There is something in the npm package that is hard coded to `christopherdro`'s paths that is causing failures on other machines and environments.

After a quick search of the repo, I can't find an obvious location but @christopherdro probably has a good idea, if he's still active on the library.

## Environment info

      

`react-native info` output:

```
System:
    OS: macOS 11.2.3
    CPU: (12) x64 Intel(R) Core(TM) i7-8700B CPU @ 3.20GHz
    Memory: 2.87 GB / 64.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 15.2.1 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.6.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: Not Found
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 15, 17, 22, 23, 25, 26, 27, 28, 29
      Build Tools: 26.0.0, 27.0.3, 28.0.0, 28.0.0, 28.0.0, 28.0.2, 28.0.3, 29.0.2, 29.0.3
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7199119
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_111 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.0 => 0.64.0 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

```bash
react-native: 0.64.0
      react: 17.0.1
      react-native-maps: 0.28.0
```

## Steps to reproduce

1. Remove `node_modules` with `rm -rf node_modules`
2. Ensure `react-native-maps` is added to `package.json`.
3. Install packages with `yarn install`.
4. Try building an APK with `gradlew assembleRelease`.
5. Encounter error related to @christopherdro's user path, as shown above.
6. Clean assets with `gradlew clean`.
7. Retry building an APK with `gradlew assembleRelease`.
8. Success.

Describe what you expected to happen:

APK is built without encountering an error related to a hard coded path for a different user and their machine/environment.

## Reproducible sample code

See above for steps on installation to reproduce.

**Comment by drmusicforever** [comment]
Having the same exact issue 

```
* What went wrong:
Execution failed for task ':react-native-maps:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > 1 exception was raised by workers:
     com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
     AAPT: /Users/christopherdro/Desktop/OSS/react-native-maps/lib/android/build/intermediates/res/merged/release/layout/notification_template_big_media_custom.xml:63: error: resource style/TextAppearance.Compat.Notification.Time.Media (aka com.airbnb.android.react.maps:style/TextAppearance.Compat.Notification.Time.Media) not found.

```

**Comment by joshuapinter** [comment]
@drmusicforever Okay good. Glad I'm not the only one. Thanks for posting.

**Comment by cchiaramelli** [comment]
Same here

**Comment by joshuapinter** [comment]
![dozens_of_us_arrested_development](https://user-images.githubusercontent.com/180819/117091011-2fa30d00-ad17-11eb-9205-ea6bef8d131c.gif)

**Comment by joshuapinter** [comment]
@christopherdro Why did you close this? Has it been resolved?

**Comment by itsravenous** [comment]
I still experience this issue on 0.28.0 - `gradlew clean` isn't resolving it for me either...

**Comment by joshuapinter** [comment]
> I still experience this issue on 0.28.0 - ~`gradlew clean` isn't resolving it for me either...~ the clean _does_ solve it.

Thanks for correcting that. I was curious why that wouldn't work for you. 

But this still should be fixed properly and I'm not sure why it was closed without a proper resolution.

**Comment by fokoz** [comment]
@christopherdro This issue should been fixed permanently, it is not quite convenience to clean for every builds. Thank you.

**Comment by christopherdro** [comment]
You shouldn't have to clean for every single build but the issue stems from the build directories being included in the npm package. This has been resolved via https://github.com/react-native-maps/react-native-maps/pull/3810 and is included in v0.29.0.

**Comment by joshuapinter** [comment]
Thanks for fixing this, @christopherdro! We'll upgrade and let you know if there's any trouble.
